src/main/java/org/apache/commons/compress/compressors/brotli/BrotliUtils.java

```diff
@@ -72,7 +72,7 @@ private static boolean internalIsBrotliCompressionAvailable() {
      * <p>This defaults to {@code false} in an OSGi environment and {@code true} otherwise.</p>
      * @param doCache whether to cache the result
      */
-    public static void setCacheBrotliAvailablity(final boolean doCache) {
+    public static void setCacheBrotliAvailability(final boolean doCache) {
```

Comment:
this change breaks backwards compatibility,

Reply:
> this change breaks backwards compatibility,

done.